devops/actions/e2e-tests-win/action.yml

```diff
@@ -30,7 +30,7 @@ runs:
     run: |
       mkdir build-e2e
       set PATH=%GITHUB_WORKSPACE%\install\bin;%PATH%
-      cmake -GNinja -B build-e2e -S.\llvm\sycl\test-e2e -DSYCL_TEST_E2E_TARGETS="${{ inputs.targets }}" -DCMAKE_CXX_COMPILER="clang++" -DLLVM_LIT="..\llvm\llvm\utils\lit\lit.py" ${{ inputs.cmake_args }}
+      cmake -GNinja -B build-e2e -S.\llvm\sycl\test-e2e -DSYCL_TEST_E2E_TARGETS="${{ inputs.targets }}" -DCMAKE_CXX_COMPILER="clang++" -DLEVEL_ZERO_LIBS_DIR="D:\github\level-zero_win-sdk\lib.fake" -DLLVM_LIT="..\llvm\llvm\utils\lit\lit.py" ${{ inputs.cmake_args }}
```

Comment:
I'm not sure if those changes are actually tested by pre-commit CI. You might need to push your own branch into the repo and modify the pre-commit tasks not to skip on that branch. After that a test PR against your newly pushed branch could actually test this change.

Reply:
I specially used "fake" location to check that interop tests fail, so I know it is tested.